test/runtests.jl

```diff
@@ -164,7 +176,7 @@ end
     end
 
     if !is_APPVEYOR && GROUP == "AlgConvergence_II"
-        @time @safetestset "SSPRK Tests" include("algconvergence/ode_ssprk_tests.jl")
+        @time @safetestset "SSPRK Tests" include("../lib/OrdinaryDiffEqSSPRK/test/ode_ssprk_tests.jl")
```

Comment:
split groups

Reply:
![image](https://github.com/SciML/OrdinaryDiffEq.jl/assets/128291516/6b8a3175-c804-41d2-a125-e48373468764)

This way ??

Reply:
Yes, I think I forgot to check that on the last PR too

Reply:
I didn't split groups in the last PR. I will do it in this one